html-syntax/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Html Syntax</title>
+  </head>
+  <body>
+    <img src="https://cdn.bulbagarden.net/upload/thumb/8/8a/483Dialga.png/1200px-483Dialga.png" alt="Legendary Pokemon Dialga">
+    <h1>Dialga</h1>
+    <h3>&copy Pokémon.  &reg Nintendo.</h3>
```

Comment:
You should end each HTML entity with a semicolon for complete syntax. For instance, `&copy;`